BetterMD/parse/markdown.py

```
@@ -0,0 +1,356 @@
+
+class MDParser:
+
+    top_level_tags = {
+        "blockquote": r"^> (.+)$", # Blockquote
+        "br": r"\n\n", # Br
+        "code": r"^```([A-Za-z]*)[^.](?:([^`]*)[^.])?```$", # Code block
+
+        "h": r"^(#{1,6})(?: (.*))?$",
+
+        "hr": r"^---+$", # Hr
+        "br": r"\s", # Br
```

Ruff (0.8.2)

Dictionary key literal `"br"` repeated

(F601)